source/administration/command-line-tools.rst

```diff
@@ -0,0 +1,661 @@
+
+Deactivate a user. Deactivated users are immediately logged out of all
+sessions and are unable to log back in.
+
+::
+
+    platform user deactivate [emails, usernames, userIds]
+
+**Examples**
+
+::
+
+      user deactivate user@example.com
+      user deactivate username
+
+platform user delete
+^^^^^^^^^^^^^^^^^^^^
+
+Permanently deletes a user and all related information, including posts.
+
+::
+
+    platform user delete [users]
+
+**Examples**
+
+::
+
+      user delete user@example.com
+
+**Options**
+
+::
+
+          --confirm   Confirm you really want to delete the user and a DB backup has been performed.
+
+platform user deleteall
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Permanently delete all users and all related information, including
+posts.
+
+::
+
+    platform user deleteall
+
+**Examples**
+
+::
+
+      user deleteall
+
+**Options**
+
+::
+
+          --confirm   Confirm you really want to delete the user and a DB backup has been performed.
+
+platform user invite
+^^^^^^^^^^^^^^^^^^^^
+
+Send a user an email invite to a team. You can invite a user to multiple
+teams by listing the team names or team IDs.
+
+::
+
+    platform user invite [email] [teams]
+
+**Examples**
+
+::
+
+      user invite user@example.com myteam
+      user invite user@example.com myteam1 myteam2
+
+platform user migrate\_auth
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Migrates all user accounts from one authentication provider to another.
+For example, you can upgrade your authentication provider from email to
+ldap. Output will display any accounts that are not migrated
+successfully.
+
+-  ``from_auth``: The authentication service to migrate users accounts
+   from. Supported options: ``email``, ``gitlab``, ``saml``.
+
+-  ``to_auth``: The authentication service to migrate users to.
+   Supported options: ``ldap``.
+
+-  ``match_field``: The field that is guaranteed to be the same in both
+   authentication services. For example, if the users emails are
+   consistent set to email. Supported options: ``email``, ``username``.
+
+::
+
+    platform user migrate_auth [from_auth] [to_auth] [match_field]
+
+**Examples**
+
+::
+
+      user migrate_auth email ladp email
+
+platform user password
+^^^^^^^^^^^^^^^^^^^^^^
+
+Set a user's password.
+
+::
+
+    platform user password [user] [password]
+
+**Examples**
+
+::
+
+      user password user@example.com Password1
+
+platform user resetmfa
+^^^^^^^^^^^^^^^^^^^^^^
+
+Turns off multi-factor authentication for a user. If MFA enforcement is
+enabled, the user will be forced to re-enable MFA with a new device as
+soon as they log in.
+
+::
+
+    platform user resetmfa [users]
+
+**Examples**
+
+::
+
+      user resetmfa user@example.com
+
+platform user verify
+^^^^^^^^^^^^^^^^^^^^
+
+Verify the email address of a new user.
+
+::
+
+    platform user verify [users]
+
+**Examples**
+
+::
+
+      user verify user1
+
+platform version
+~~~~~~~~~~~~~~~~
+
+Displays Mattermost version information.
+
+::
+
+    platform version
```

Comment:
![image](https://cloud.githubusercontent.com/assets/20325173/21911602/54f5470e-d8d6-11e6-86bd-4b493521e83d.png)

Suggest using a markup like this:
```
platform channel add
^^^^^^^^^^^^^^^^^^^^

  Description
    Add users to a channel.

  Format
    platform channel add *[channel]* *[users]*

  Example
    ``./platform channel add mychannel user@example.com username``

platform channel create
^^^^^^^^^^^^^^^^^^^^^^^

  Description
    Create a channel.

  Format
    platform channel create *[options]*

  Options
    ``--display_name [channel-display-name]`` 
    
    ``--header [channel-header]``
    
    ``--name [channel-name]``
    
    ``--private``
    
    ``--purpose [channel-purpose]``
    
    ``--team [team-name-or-id]``

  Examples
    ``./platform channel create --team "myteam" --name "newchannel" --display_name "My New Channel"``
    
    ``./platform channel create --team "myteam" --name "privatechannel" --display_name "Private Channel" --private``

```
Then the output looks like this:

![image](https://cloud.githubusercontent.com/assets/20325173/21911714/d8c6ec18-d8d6-11e6-91d8-9fd9379efb6a.png)